src/plugins/workspace/public/components/workspace_creator/workspace_creator.tsx

```diff
@@ -214,6 +214,7 @@ export const WorkspaceCreator = (props: WorkspaceCreatorProps) => {
           paddingSize="none"
           color="subdued"
```

Comment:
```suggestion
          color="transparent"
```
We can change it here if we want to make the background transparent.